README.md

```diff
@@ -194,6 +194,19 @@ To run the application locally you will need:
 
 1. Run `cp example.env .env`
 
+Note that the local static JS and CSS files have to be installed into place 
```

Comment:
I think since this in already in the "running the app" section, we should just give our preferred steps, which'd be to run the commands rather than commenting out volumes.

So I'd lay it out something like:

1. Run `cp example.env .env`
2. Run` npm ci`
3. Run `npm run collectjs && npm run collectcss`
4. Run `docker compose up`

```

Reply:
note for future, this is something we typically use a [`bin/setup`](https://github.com/ackama/rails-template/blob/main/variants/backend-base/bin/setup.tt) script to help with

Reply:
Fair!